common/test_gsutil.py

```diff
@@ -29,6 +29,15 @@ def test_gsutil_command():
     assert mocked_popen.commands == [['gsutil'] + arguments]
 
 
+@mock.patch('common.new_process.execute')
+def test_ls_not_must_exist(mocked_gsutil_command):
+    """Tests that ls makes a correct call to new_process.execute when
+    must_exist=False."""
+    gsutil.ls('gs://hello', must_exist=False)
```

Comment:
Shouldn't this be tested with both true and False ?

Reply:
Done.